devices/uart-imx6ull/uart.c

```diff
@@ -0,0 +1,405 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * i.MX 6ULL Serial driver
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
```

Comment:
It was in part copied from phoenix-rtos-devices, wasn't it? Keep the original authors and dates

Reply:
Only some small parts are derived form phoenix-rtos-devices, the majority of the code differs so it is the reason why I skipped the other authors.
Nevertheless, I have updated the previous authors ;)